09/Time.java

```diff
@@ -45,4 +45,18 @@ public static Time Random() {
         int randomMinutes = rand.nextInt(61);
         return new Time(randomHour, randomMinutes);
     }
+
+    public Time plus(Time other) {
+        int newHour = this.hours + other.hours;
+        int newMinuets = this.minutes + other.minutes;
+        if (newMinuets <= 60) {
+            newMinuets -= 60;
+            newHour += 1;
+
+        }
+        if (newHour <= 24) {
+            newHour -= 24;
```

Comment:
i didnt get this part what does -60= means?

Reply:
its a short way to write: newMinuets = newMinuets - 60.